packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,931 @@
+
+#### Org-wide CAPTCHA settings properties
+
+| Property           | Type                           | Description                                                                                                       |
+| ------------------ | ------------------------------ | ----------------------------------------------------------------------------------------------------------------- |
+| `captchaId`        | String                         | Required. Unique key of assigned CAPTCHA|
+| `enabledPages`     | `SSR`, `SSPR`,`SIGN_IN`        | Required. Array of CAPTCHA supported page.|
+
+Explain for available values in property `enabledPages`:
+
+| CAPTCHA Support Page | Description |
+| -------------------- | ----------- |
+| `SSR` | Self-service Registration |
+| `SSPR`| Self-service Password Recovery |
+| `SIGN_IN`| User login page |
+
+#### Org-wide CAPTCHA settings example
+
+```json
+{
+  "captchaId": "cap18c2Ey3iR9BLDb0g4",
+  "enabledPages": ["SSPR", "SIGN_IN", "SSR"]
+}
+```
+
+### Org-wide CAPTCHA settings Response object
+
+#### Org-wide CAPTCHA settings Response properties
+Property           | Type                             | Description                                                                                                       |
+| ------------------ | ------------------------------ | ----------------------------------------------------------------------------------------------------------------- |
+| `captchaId`        | String                         | Required. Unique key of assigned CAPTCHA|
+| `enabledPages`     | `SSR`, `SSPR`,`SIGN_IN`        | Required. Array of CAPTCHA supported page.|
+| `_links`           | [Links](#org-wide-captcha-links-object) | Link relations for this object|
+
+#### CAPTCHA example
+```json
+{
+    "enabledPages": [
+        "SSPR"
+    ],
+    "captchaId": "cap18c2Ey3iR9BLDb0g4",
+    "_links": {
+        "captcha": {
+            "href": "http://testorg.okta1.com:1802/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "POST",
+                    "PUT",
+                    "DELETE"
+                ]
+            }
+        },
+        "self": {
+            "href": "http://testorg.okta1.com:1802/api/v1/org/captcha",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
+### Org-wide CAPTCHA Links object
+
+Specifies link relations (see [Web Linking](http://tools.ietf.org/html/rfc8288)) available for the current status of an application using the [JSON Hypertext Application Language](http://tools.ietf.org/html/draft-kelly-json-hal-06) specification. This object is used for dynamic discovery of related resources and lifecycle operations. The Links object is read-only.
+
+| Type | Description                                                                                     |
```

Comment:
I think the links are pretty self-explanatory, so this table isn't necessary.

Reply:
I will remove it from doc